### PR TITLE
feat(lib): expose operators on their own API class

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

File: packages/cdktf/lib/terraform-operators.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import { Expression, OperatorExpression } from "./tfExpression";
+import { IResolvable } from "./tokens/resolvable";
+
+/**
+ * This class contains static functions for all arithmetical and logical operators
+ * in the Terraform configutation language.
+ */
+export class Op {
+  /**
+   * Renders !expr
```
Comment (ansgarm) [inline packages/cdktf/lib/terraform-operators.ts:10]: nit: shall we match the name of the argument?
```suggestion
   * Renders !expression
```

File: packages/cdktf/lib/terraform-operators.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  /**
+   * Renders expr1 + expr2
+   */
+  public static add(left: Expression, right: Expression): IResolvable {
+    return new OperatorExpression("+", left, right);
+  }
+
+  /**
+   * Renders expr1 - expr2
```
Comment (ansgarm) [inline packages/cdktf/lib/terraform-operators.ts:52]:
```suggestion
   * Renders left - right
```

Comment (ansgarm) [reply packages/cdktf/lib/terraform-operators.ts:52]: same for other comments

**Review by ansgarm — APPROVED** [review]
Nice!
Just a small set of nits regarding the code comments 😇

**Review by mutahhir — APPROVED** [review]
Nice! ❤️

**Review by laurapacilio — APPROVED** [review]
I assume the API docs are autogenerated? So I'm assuming it doesn't make sense to really review them here :). Otherwise, hooray - nice work!

**Comment by ansgarm** [comment]
> I assume the API docs are autogenerated?

@laurapacilio: They are based off of the comments in this file: https://github.com/hashicorp/terraform-cdk/blob/main/packages/cdktf/lib/terraform-operators.ts

**Pull request merged**